tests/upgrades/test_virtwho.py

```diff
@@ -0,0 +1,195 @@
+"""Test for Virt-who related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
+from fauxfactory import gen_string
+from nailgun import entities
+from robottelo.decorators import skip_if_not_set
+from robottelo.test import APITestCase, settings
+from robottelo.virtwho_utils import (
+    deploy_configure_by_command,
+    get_configure_command,
+    get_configure_file,
+    get_configure_option,
+    get_hypervisor_info
+)
+from upgrade_tests import pre_upgrade, post_upgrade
+from wait_for import wait_for
+
+
+class scenario_positive_virt_who(APITestCase):
+    """Virt-who config is intact post upgrade and verify virt-who update/delete
+
+    :steps:
+
+        1. In Preupgrade Satellite, create virt-who-config
+        2. Upgrade the satellite to next/latest version
+        3. Postupgrade, Verify the virt-who config is intact, update and delete
+
+    :expectedresults: Virtwho config should create, update and delete successfully.
```

Comment:
grammar: "should be created, updated and deleted successfully"?

Reply:
yes, thanks